ansible/playbooks/roles/upgrade/tasks/postgres-exporter.yml

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
For this release we started moving new code for upgrades under the specific roles themselves so we don't need to rewrite some of that stuff later when moving to the collections. Check for example:

/epiphany/ansible/playbooks/roles/prometheus/tasks/upgrade ...

As this is a new upgrade I suggest we do the same.